Services/CO.CDP.Organisation.WebApi/UseCase/GetReviewsUseCase.cs

```diff
@@ -0,0 +1,15 @@
+using AutoMapper;
+using CO.CDP.Organisation.WebApi.Model;
+using CO.CDP.OrganisationInformation.Persistence;
+
+namespace CO.CDP.Organisation.WebApi.UseCase;
+public class GetReviewsUseCase(IOrganisationRepository organisationRepository, IMapper mapper) : IUseCase<Guid, IEnumerable<Model.Review>>
+{
+    public async Task<IEnumerable<Model.Review>> Execute(Guid organisationId)
+    {
+        var organisation = await organisationRepository.Find(organisationId)
+                           ?? throw new UnknownOrganisationException($"Unknown organisation {organisationId}.");
+
+        return [ mapper.Map<Review>(organisation) ];
```

Comment:
If Review is not a collection probably just remove collection from method response type

Reply:
The idea here was to expose a collection of one item _now_, so that when the reviews API is extended later then we don't need to change the way we consume it from the frontend. We are anticipating that very soon we will need to have multiple reviews per org.

But, I do take your point. I don't know - what do you think @jakzal ?

Reply:
Keep the collection, please. We've done this consciously, as we anticipate we will need multiple reviews per org. 

At the moment, reviewers can add reviews multiple times, but we always overwrite the previous review.